test/page-detect.ts

```diff
@@ -1,21 +1,53 @@
-import test, {ExecutionContext} from 'ava';
+import test from 'ava';
 import './fixtures/globals';
 import * as pageDetect from '../source/libs/page-detect';
 
-function urlMatcherMacro(
-	t: ExecutionContext,
-	detectFn: (url: string) => boolean,
-	shouldMatch: string[] = [],
-	shouldNotMatch: string[] = []
-): void {
-	for (const url of shouldMatch) {
-		location.href = url;
-		t.true(detectFn(url));
+const allUrls = new Set<string>();
+for (const imported of Object.values(pageDetect)) {
+	if (Array.isArray(imported)) {
+		for (const url of imported) {
+			allUrls.add(url);
+		}
+	}
+}
+
+for (const [key, detect] of Object.entries(pageDetect)) {
+	if (key.endsWith('Test') || typeof detect !== 'function') {
```

Comment:
```suggestion
	if (key.endsWith('Test') || key.endsWith('TestSkipNegatives') || typeof detect !== 'function') {
```

Reply:
It already matches `typeof detect !== 'function'` because in this case `detect` will be `true`.

If this wasn't the case, it would error later anyway with:

```
The function `isRepoTestSkipNegatives` doesn’t have any tests. Export an array of valid URLs as `isRepoTestSkipNegativesTest`
```

Reply:
Ah, yes. Ignore my suggestion.